.github/workflows/stm32build.yml

```diff
@@ -69,6 +69,20 @@ jobs:
           echo "Test failed"
           exit 1
         fi
+  run_static_tests:
```

Comment:
An alternative is to run the static tests as an extra step of run_tests, so the already running environments with the checked out code is used.